aiida/tools/importexport/archive/migrations/__init__.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Migration archive files from old export versions to the newest, used by `verdi export migrate` command."""
+from pathlib import Path
+from typing import Any, Callable, Dict, Tuple, Union
+
+from aiida.common.folders import Folder
+
+from .v01_to_v02 import migrate_v1_to_v2
+from .v02_to_v03 import migrate_v2_to_v3
+from .v03_to_v04 import migrate_v3_to_v4
+from .v04_to_v05 import migrate_v4_to_v5
+from .v05_to_v06 import migrate_v5_to_v6
+from .v06_to_v07 import migrate_v6_to_v7
+from .v07_to_v08 import migrate_v7_to_v8
+from .v08_to_v09 import migrate_v8_to_v9
+
+# version from -> version to, func(path to (uncompressed), dict of cached data) -> dict of cached data
```

Comment:
```suggestion
# version from -> version to, func(path to uncompressed json, dict of cached data) -> dict of migrated data
```

Reply:
However, this doesn't really do this function justice, since what matters is the side effect of the migrated data being written to disk, right?

Reply:
yep